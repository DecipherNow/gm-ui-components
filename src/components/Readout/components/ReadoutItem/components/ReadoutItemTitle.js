import styled from "styled-components";
import { keen } from "style/theme";

const ReadoutItemTitle = styled.h2`
  font-size: ${({ theme }) => theme.FONT_SIZE_TEXT_DEFAULT};
  margin: 0;
  padding: 0;
  font-weight: ${({ theme }) => theme.FONT_WEIGHT_SEMIBOLD};
  min-height: 1em;
  display: inline-block;
`;

ReadoutItemTitle.defaultProps = {
  theme: keen
};

export default ReadoutItemTitle;
