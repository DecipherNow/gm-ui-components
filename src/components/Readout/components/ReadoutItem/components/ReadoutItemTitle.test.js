import React from "react";
import { shallow } from "enzyme";
import ReadoutItemTitle from "./ReadoutItemTitle";

describe("ReadoutItemTitle", () => {
  it("matches snapshot", () => {
    const aReadoutItemTitle = shallow(<ReadoutItemTitle />);
    expect(aReadoutItemTitle).toMatchSnapshot();
  });
});
