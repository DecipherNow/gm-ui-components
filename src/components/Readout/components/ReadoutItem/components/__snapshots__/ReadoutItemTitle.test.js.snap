// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadoutItemTitle matches snapshot 1`] = `
.c0 {
  margin: 0;
  padding: 0;
  min-height: 1em;
  display: inline-block;
}

<h2
  className="c0"
/>
`;
